src/components/salesEvidenceSearch.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { ChangeEvent } from 'react'
+import { IntersectionFilterParams } from '../types'
+import { SearchWidget } from './SearchWidget'
+import DatePicker from 'react-datepicker'
+import 'react-datepicker/dist/react-datepicker.css'
+import { SaleTypeDropdown } from './SaleTypeDropdown'
+
+type SearchWidgetProps = {
+    searchParameters: IntersectionFilterParams
+    changeSearchParameters: (newParameters: IntersectionFilterParams) => void
+}
+
+export function SalesEvidenceSearchWidget (props: SearchWidgetProps) {
+    return (
+        <>
+            <SearchWidget changeSearchParameters={props.changeSearchParameters} searchParameters={props.searchParameters} />
+            <form className="wrapper">
+                <label className="eight">Land Area m2<br />
+                    <input className="minMaxSize border" id="landAreaMin" value={props.searchParameters.landArea.min} onChange={(e: ChangeEvent<HTMLInputElement>) => {
+                        props.changeSearchParameters({
+                            ...props.searchParameters,
+                            landArea: { min: e.target.valueAsNumber, max: props.searchParameters.landArea.max }
+                        })
+                    }} placeholder="Enter min land area." type="number" tabIndex={6} />
+                    <input className="minMaxSize border" id="landAreaMax" value={props.searchParameters.landArea.max} onChange={(e: ChangeEvent<HTMLInputElement>) => {
+                        props.changeSearchParameters({
+                            ...props.searchParameters,
+                            landArea: { max: e.target.valueAsNumber, min: props.searchParameters.landArea.min }
```

Comment:
It's preferable (at least for me!) to keep a consistent order in the objects. In Land Area m2 you have `{min, max}` and in this one you have `{max, min}`. This is especially disorienting when looking at the React state and they change between objects.

So yeah, just change to `{min, max}`